week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -24,8 +33,17 @@ function exerciseOne(arrayOfPeople) {
  * All of your HTML should go inside the Div tag with the id "content".
  *
  */
+
 function exerciseTwo(shopping) {
-  //Write your code in here
+    let content = document.querySelector("#content");
+    let ul = document.createElement("ul");
+    content.appendChild(ul);
+
+    shopping.forEach((item) => {
```

Comment:
thats great. it shows that you clearly understand how to traverse through the array and adding ul and li tags to the content